src/bun.js/bindings/bindings.zig

```diff
@@ -6131,6 +6146,9 @@ pub fn toJSHostFunction(comptime Function: anytype) JSC.JSHostFunctionType {
 
 const DeinitFunction = *const fn (ctx: *anyopaque, buffer: [*]u8, len: usize) callconv(.C) void;
 
+pub const JSBuiltinFunctionType = fn (*JSGlobalObject) callconv(JSC.conv) JSValue;
```

Comment:
This name is confusing. "JS Builtins" refer to the files like `ReadableStreamInternals.ts`. Is this related to the `JS2Native` generator? If so, maybe it should say `JS2NativeFunctionType` instead.